back-end/src/index.ts

```diff
@@ -14,24 +14,65 @@ async function serverSetup() {
   app.use(bodyParser.urlencoded({ extended: true }));
 
   app.locals.db = await getDB();
+
+  // query to make a function to automatically add date everytime insert a data to db
```

Comment:
I think it's better to use .NOW() function when inserting to the tables (forumModel)  instead of using this at index.ts

Reply:
And also remove unnecessary comments to make the code keep clean

Reply:
> I think it's better to use .NOW() function when inserting to the tables (forumModel) instead of using this at index.ts

what do you mean by this? i'm not quite getting it. the reason i'm making it in index.ts is so you don't have to insert the date on forumModel, let the query automatically insert the date

Reply:
and that function is used when the data is updated. so i make 2 type of dates created date and updated  date. when the data is updated the updated date is changed to the time it was updated